Exiled.Permissions/Extensions/Permissions.cs

```diff
@@ -172,11 +172,18 @@ public static void Reload()
         /// <returns>Returns a value indicating whether the user has the permission or not.</returns>
         public static bool CheckPermission(this CommandSender sender, string permission)
         {
-            if (sender.FullPermissions || sender is ServerConsoleSender)
+            if (sender.FullPermissions || sender is ServerConsoleSender || sender is GameCore.ConsoleCommandSender || sender == Server.Host.Sender)
+            {
                 return true;
-            else if (sender is PlayerCommandSender && Player.Get(sender.SenderId) is Player player)
-                return player == Server.Host || player.CheckPermission(permission);
+            }
+            else if (sender is PlayerCommandSender)
```

Comment:
```suggestion
            else if (sender is PlayerCommandSender || sender is UserPrint)
```
Should this check be added to other ifs of `Exiled.Permissions` as well? `UserPrint` is the client-side player's console.